webpack.config.babel.js

```diff
@@ -2,17 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import path from "path";
 import webpack from "webpack";
 import combineLoaders from "webpack-combine-loaders";
 import CopyWebpackPlugin from "copy-webpack-plugin";
 import ExtractTextPlugin from "extract-text-webpack-plugin";
-import HtmlWebpackPlugin from "html-webpack-plugin";
+import HTMLWebpackPlugin from "html-webpack-plugin";
+import XMLWebpackPlugin from "xml-webpack-plugin";
 import MinifyPlugin from "babel-minify-webpack-plugin";
-import path from "path";
 
-const entries = {
-  "webextension/background": "./webextension/background/index.js",
-  "webextension/manage/index": "./webextension/manage/index.js",
+import JSONWebpackPlugin from "./json-webpack-plugin";
+import thisPackage from "./package.json";
+
+const addonInfo = {
+  name: thisPackage.title,
+  id: "lockbox@mozilla.com",
```

Comment:
this can also be in the package.json, as `extensionId` since `id` seems too generic.

Reply:
`id` might be ok actually, since it's what `jpm` uses when it generates `install.rdf` from `package.json`...